kibana/Dockerfile

```diff
@@ -4,31 +4,22 @@ MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 EXPOSE 5601
 
-ENV KIBANA_VER=4.1.2 \
+ENV KIBANA_VER=4.5.1 \
```

Comment:
@ewolinetz  Need to bump to 4.5.4 to fix ESA-2016-04 and ESA-2016-03 identified here: https://www.elastic.co/community/security